### DISCUSSION TITLE
New upgrades

### DISCUSSION THREAD

**Discussion by olivervincent** [BG-Software-LLC/SuperiorSkyblock2#1224] [Denied Feature Requests] [answered]
Hey @OmerBenGera ,

I tried to create the island chest upgrades, as you said.

Here is what I created:
https://pastebin.com/JwA64NAM

It's only the first 3 levels. The menu loads in fine, and the first level, goes in and execute the level 1 command fine. But it doesn't go to the next level? I can just spam the upgrade button: https://i.imgur.com/sz49KFD.png https://i.imgur.com/rNSwKQh.png

What am I doing wrong? xd

**Comment by OmerBenGera** [comment, marked as answer]
Follow this guide: https://wiki.bg-software.com/superiorskyblock/overview/upgrades

Do not open discussions for seeking help - use the discord server instead.
